app/client/src/routes/home.tsx

```diff
@@ -1,16 +1,511 @@
+  }, [dontShowAgain]);
+
+  if (content.status === 'pending') return <Loading />;
+
+  if (content.status === 'failure')
+    return (
+      <Alert type="error">
+        Expert Query is currently unavailable, please try again later.
+      </Alert>
+    );
+
+  if (content.status === 'success')
+    return (
+      <div>
+        {introVisible && (
+          <Summary heading="How to Use This Application">
+            <p>
+              Select a data profile, then build a query by selecting options
+              from the input fields.
+            </p>
+            <div className="display-flex flex-justify flex-wrap">
+              <Checkbox
+                checked={dontShowAgain ?? false}
+                label="Don't show again on this computer"
+                onChange={(_ev) => setDontShowAgain(!dontShowAgain)}
+                styles={['margin-right-1 margin-y-auto']}
+              />
+              <Button
+                onClick={() => setIntroVisible(false)}
+                styles={['margin-top-2']}
+              >
+                Close Intro
+              </Button>
+            </div>
+          </Summary>
+        )}
+        <h3>Data Profile</h3>
+        <Select
+          aria-label="Select a data profile"
+          onChange={(ev) => inputDispatch({ type: 'dataProfile', payload: ev })}
+          options={listOptions.dataProfile}
+          placeholder="Select a data profile..."
+          value={dataProfile}
+        />
+        {dataProfile && (
+          <>
+            <h3>Filters</h3>
+            <FilterFields
+              dispatch={inputDispatch}
+              fields={profiles[dataProfile.value].fields}
+              options={{ ...listOptions, ...content.data.domainValues }}
+              state={inputState}
+            />
+            <h3>Download the Data</h3>
+            <div className="display-flex flex-wrap">
+              <RadioButtons
+                legend={
+                  <>
+                    <b className="margin-right-1">File Format</b>
+                    <InfoTooltip text="Choose a file format for the result set" />
+                  </>
+                }
+                onChange={(option) =>
+                  inputDispatch({ type: 'fileFormat', payload: option })
+                }
+                options={listOptions.fileFormat}
+                selected={fileFormat}
+                styles={['margin-bottom-2']}
+              />
+              <div className="display-flex flex-column flex-1 margin-y-auto">
+                <Button
+                  onClick={() => {}}
+                  styles={['margin-x-auto', 'margin-bottom-1']}
+                >
+                  Download
+                </Button>
+                <Button
+                  color="white"
+                  onClick={(_ev) => inputDispatch({ type: 'reset' })}
+                  styles={['margin-x-auto']}
+                >
+                  Clear Search
+                </Button>
```

Comment:
```suggestion
                <button
                  onClick={() => {}}
                  className="usa-button margin-x-auto margin-bottom-1"
                >
                  Download
                </button>
                <button
                  onClick={(_ev) => inputDispatch({ type: 'reset' })}
                  className="usa-button usa-button--outline margin-x-auto"
                >
                  Clear Search
                </button>
```

Part of comment above. I don't think this component is necessary at this time.